pr_agent/tools/pr_description.py

```diff
@@ -74,7 +74,7 @@ async def run(self):
 
             await retry_with_fallback_models(self._prepare_prediction)
 
-            get_logger().info(f"Preparing answer {self.pr_id}")
+            get_logger().info(f"Preparing description regarding {self.pr_id}")
```

Comment:
**Suggestion:** Consider using structured logging instead of formatted strings for better log management and analysis.
```suggestion
            get_logger().info("Preparing description regarding", extra={"pr_id": self.pr_id})
```